kork/kork-web/src/main/java/com/netflix/spinnaker/kork/client/ServiceClientProvider.java

```diff
@@ -25,7 +25,11 @@
 
 @NonnullByDefault
 public interface ServiceClientProvider {
-
+  /** Which version of retrofit to use. Retrofit 1.x and 2.x are supported. */
+  enum RetrofitVersion {
+    RETROFIT1,
```

Comment:
I don't see anything using RETROFIT1.  Can you add a test, or something so we can see how this works?

Reply:
Here it is - [RetrofitServiceProviderTest.kt](https://github.com/spinnaker/spinnaker/pull/7131/files#diff-2e4624eda13fb9fa674ea7025abb5c7ceb4d3fee76dd12ca7c4546bc9ff8fd7e)